src/index.ts

```diff
@@ -59,10 +55,27 @@ class Tox {
         })
     }
     addFriend(toxId: string, message?: string): Promise<ToxResponse> {
-        throw new Error("Unimplemented")
+        return new Promise((accept, reject) => {
+            let request: ToxRequest;
+
+            if(message !== undefined) {
+                request = { request: "AddFriend", tox_id: toxId, message }
+            }
+            else {
+                request = { request: "AddFriendNorequest", tox_id: toxId }
```

Comment:
How about an optional `message` field in AddFriend?

Reply:
I want to keep the protocol close to tox API.

Reply:
OK